recipes/_lustre_install.rb

```diff
@@ -44,4 +45,6 @@
   yum_package 'lustre_client' do
     source lustre_client_rpm
   end
+elsif node['platform'] == 'centos'
+  Chef::Log.error("Unsupported version of Centos, #{node['platform_version']}, supported versions are 7.6 and 7.5")
```

Comment:
warn message here since it's not an error for which the build should fail.

Reply:
changed